charts/presto/values.yaml

```diff
@@ -239,6 +245,19 @@ worker:
   tolerations: []
   # Worker security context (overwrites default security context)
   securityContext: {}
+  
+# Enables Presto to use Velox based workers  
+velox: 
+  enabled: false
+  # Image details for Velox workers (overrides common image details)
+  image:
+    repository: prestodb/presto-native
+    pullPolicy: IfNotPresent
+    tag: ~
+  # Velox properties
+  config: |-
```

Comment:
We've deprecated velox.properties, so let's just remove this: https://github.com/prestodb/presto/pull/21962

Reply:
sure, removed.